### PR TITLE
Port account_invoice_merge to Odoo 8.0 with new API

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

File: account_invoice_merge/wizard/invoice_merge.py [hunk 1]
```diff
@@ -18,61 +18,62 @@
...
-    def _dirty_check(self, cr, uid, context):
-        if context.get('active_model', '') == 'account.invoice':
-            ids = context['active_ids']
+    @api.model
+    def _dirty_check(self):
+        if self.env.context.get('active_model', '') == 'account.invoice':
+            ids = self.env.context['active_ids']
             if len(ids) < 2:
-                raise orm.except_orm(
+                raise exceptions.except_orm(
```
Comment (pedrobaeza) [inline account_invoice_merge/wizard/invoice_merge.py]: You should use exceptions.Warning (with only one parameter)

**Comment by pedrobaeza** [comment]
Little remark. The rest LGTM.

File: account_invoice_merge/wizard/invoice_merge.py (hunk 1 above)
Comment (Cedric-Pigeon) [inline account_invoice_merge/wizard/invoice_merge.py]: Done in last commit.

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by rdeheele** [comment]
:+1:

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza 

If everything is ok, is it possible to merge this PR ?

Thanks.

**Comment by pedrobaeza** [comment]
Yeah, we can proceed with the 2 approvals.

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza  Thank you very much !

**Pull request merged**